### ISSUE TITLE
Input json does not support symlink file while input_parameters works fine with it 

### ISSUE THREAD

**Issue by yuankunzhu** [rabix/bunny#403]
Recently noticed this. Only tested on v1.0.2 as I have the same issue as #380 with v1.0.3.

Basically before running rabix, I have
```
ln -s /path/to/file.txt data/
```
and got input.json as
```
{
  "input_file": {"class": "File", "path":"data/file.txt"}
}
```
Then, when execute rabix, I will get
- `[ERROR] Job failed with exit code 1.` if I do `rabix tool.cwl input.json`
- `[INFO] Job root has completed` if I do `rabix tool.cwl -- --input_file data/file.txt`

**Comment by milos-ljubinkovic** [comment]
Yes it seems that we have 2 different methods for path resolution, one that follows links used in resolving "--file" syntax and one that doesn't which is used everywhere else.

Not really sure that this should be changed so that every method follows links as there might be some complications with staging input files that uses links (this system should be changed soon as well).
